cancermuts_streamlit/streamlit_utils.py

```diff
@@ -0,0 +1,80 @@
+import streamlit as st
+import os
+import pandas as pd
+
+def filter_dataframe(df: pd.DataFrame) -> pd.DataFrame:
+    addFilter = st.checkbox("Add filter")
+
+    if not addFilter:
+        return df
+    
+    df = df.copy()

```

Comment:
I'm not sure why what this is meant to accomplish - you're replacing a variable with a copy of itself?

Reply:
Fixed